splunk_eventgen/lib/plugins/output/httpevent.py

```diff
@@ -182,50 +181,82 @@ def _sendHTTPEvents(self, payload):
                 self.logger.debugv("stringpayload: %s " % stringpayload)
             else:
                 self.logger.debug("Max size for payload hit, sending to splunk then continuing.")
-                try:
-                    self._transmitEvents(stringpayload)
-                    totalbytessent += len(stringpayload)
-                    currentreadsize = 0
-                    stringpayload = targetline
-                except Exception as e:
-                    self.logger.exception(str(e))
-                    raise e
-        else:
-            try:
-                totalbytessent += len(stringpayload)
-                self.logger.debug(
-                    "End of for loop hit for sending events to splunk, total bytes sent: %s ---- out of %s -----" %
-                    (totalbytessent, totalbytesexpected))
                 self._transmitEvents(stringpayload)
-            except Exception as e:
-                self.logger.exception(str(e))
-                raise e
+                totalbytessent += len(stringpayload)
+                currentreadsize = 0
```

Comment:
the `currentreadsize` should equal to `targetlinesize`? since `targetline` is not sent here.